<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfazMercado</class>
 <widget class="QDialog" name="InterfazMercado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(228, 231, 243);</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>721</width>
     <height>291</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Arbol</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ubicaci√≥n</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Frutos en el arbol</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Monto Total</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Vendidos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Perdidos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="bComprarABB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/arbol/binario_ordenado/Images/ABB/ABB_00.png</normaloff>:/arbol/binario_ordenado/Images/ABB/ABB_00.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelPoseeABB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Posee: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Arbol Binario</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDinero">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Dinero: $500</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Heap</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPoseeHeap">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Posee: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="bComprarHeap">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/arbol/heap/Images/HEAP/HEAP_00.png</normaloff>:/arbol/heap/Images/HEAP/HEAP_00.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AVL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPoseeAVL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>490</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Posee: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="bComprarAVL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/arbol/avl/Images/AVL/AVL_03.png</normaloff>:/arbol/avl/Images/AVL/AVL_03.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bonus</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPoseeBonus">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Posee: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="bComprarBonus">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/arbol/bonus/Images/BONUS/BonusTree_03.png</normaloff>:/arbol/bonus/Images/BONUS/BonusTree_03.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Espanta Pajaros</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPoseeEspanta">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Posee: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="bComprarEspanta">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/misc/Images/Scarecrow.png</normaloff>:/misc/Images/Scarecrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bVenderTodosFrutos">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Vender todos los frutos</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCostoABB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costo: $100</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCrecimientoABB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Crece: 200 seg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Precio de frutos</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrecioFrutosABB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>$0 - $5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrecioFrutosHeap">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>590</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>$0 - $5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCrecimientoHeap">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Crece: 150 seg</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCostoHeap">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>510</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costo: $130</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>570</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Precio de frutos</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrecioFrutosAVL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>590</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>$2 - $20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCrecimientoAVL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Crece: 500 seg</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCostoAVL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costo: $350</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>570</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Precio de frutos</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrecioFrutosBonus">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>590</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>$5 - $50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCrecimientoBonus">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Crece: 500 seg</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCostoBonus">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>510</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costo: $600</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>570</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Precio de frutos</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCostoEspanta">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>La cosecha se da en frutos por segundo </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelCosechaABB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cosecha: 1/20</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCosechaHeap">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cosecha: 2/35</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCosechaAVL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>550</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cosecha: 1/60</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCosechaBonus">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>550</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cosecha: 2/60</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrecioActualABB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Precio Actual: $5</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrecioActualHeap">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>610</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Precio Actual: $5</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrecioActualAVL">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>610</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Precio Actual: $20</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrecioActualBonus">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>610</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Precio Actual: $50</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCantidadMaxEspanta">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>510</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad Maxima:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
